lib/python/qmk/cli/doctor.py

```diff
@@ -154,14 +151,16 @@ def is_executable(command):
         return False
 
     # Make sure the command can be executed
-    check = run([command, '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=5, universal_newlines=True)
+    version_arg = ESSENTIAL_BINARIES[command].get('version_arg', '--version')
+    check = run([command, version_arg], stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=5, universal_newlines=True)
```

Comment:
```suggestion
    check = run([command, version_arg], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, timeout=5, universal_newlines=True)
```

Since @fauxpark found out `avrdude` prints the help to stderr.

Reply:
And `dfu-programmer`.